controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -250,6 +251,17 @@ func (r *StorageConsumerReconciler) reconcileCephClientHealthChecker() error {
 }
 
 func (r *StorageConsumerReconciler) verifyCephResourcesDoNotExist() bool {
+
+	storageClassClaimList := &ocsv1alpha1.StorageClassClaimList{}
+	storageClassClaimListOption := []client.ListOption{
+		client.InNamespace(r.namespace),
+		client.MatchingLabels(map[string]string{ConsumerUUIDLabel: string(r.storageConsumer.UID)}),
+	}
+
+	if err := r.list(storageClassClaimList, storageClassClaimListOption); err == nil && len(storageClassClaimList.Items) > 0 {
+		return false
+	}
+
 	for _, cephResource := range r.storageConsumer.Status.CephResources {
 		switch cephResource.Kind {
 		case "CephClient":
```

Comment:
No need for this manual verification we agreed to use k8s ownership-based deleteion - please see the similar comment on the storageclasscalim controller for more details.

Reply:
removed